analytics/management/commands/update_analytics_tables.py

```diff
@@ -0,0 +1,69 @@
+from argparse import ArgumentParser
+from datetime import timedelta
+
+from django.core.management.base import BaseCommand
+from django.utils import timezone
+
+from analytics.lib.data_collectors import AnalyticsStat, process_count
+from analytics.models import RealmCount, UserCount, StreamCount, DeploymentCount, PMCount, HuddleCount
+from zerver.lib.timestamp import string_to_datetime as sd, datetime_to_string as ds
```

Comment:
I think it's more readable (and better for git grep, etc) to leave these unabbreviated